myokit/formats/sbml/_parser.py

```diff
@@ -0,0 +1,1206 @@
+                    ' No unit kind provided.')
+            myokitUnit = sbml_to_myokit_unit_dict[kind]
+            myokitUnit *= float(baseUnit.get('multiplier', default=1.0))
+            myokitUnit *= 10 ** float(baseUnit.get('scale', default=0.0))
+            myokitUnit **= float(baseUnit.get('exponent', default=1.0))
+
+            # "add" composite unit to unit definition
+            unit_def *= myokitUnit
+
+        return unit_def
+
+    def _get_units(self, parameter, user_unit_dict):
+        """
+        Returns :class:myokit.Unit expression of the unit of a parameter.
+        """
+        unit = parameter.get('units')
+        if unit in user_unit_dict:
+            return user_unit_dict[unit]
+        elif unit in sbml_to_myokit_unit_dict:
+            return sbml_to_myokit_unit_dict[unit]
+        else:
+            return None
+
+    def _get_substance_units(self, species, user_unit_dict):
+        """
+        Returns :class:myokit.Unit expression of the unit of a species.
+        """
+        # Convert substance unit into myokiy.Unit
+        unit = species.get('substanceUnits')
+        if unit in user_unit_dict:
+            return user_unit_dict[unit]
+        elif unit in sbml_to_myokit_unit_dict:
+            return sbml_to_myokit_unit_dict[unit]
+        else:
+            return None
+
+    def _get_species_initial_value_in_amount(self, species, compId):
+        """
+        Returns the initial value of a species either in amount or
+        concentration depend on the flag is Amount.
+        """
+        amount = species.get('initialAmount')
+        if amount:
+            return amount
+        conc = species.get('initialConcentration')
+        if conc:
+            volume = param_and_species_dict[compId]
+            return myokit.Multiply(
+                myokit.Number(amount), myokit.Name(volume))
+        return None
+
+
+class SBMLError(myokit.ImportError):
+    """
+    Thrown if an error occurs when importing SBML
+    """
+
+
+# SBML base units according to libSBML
+sbml_to_myokit_unit_dict = {
+    'ampere': myokit.units.A,
+    'avogadro': myokit.units.mol,  # This is a simplification of SBML's def
+    'becquerel': myokit.units.Bq,
+    'candela': myokit.units.cd,
+    'celsius': myokit.units.C,
+    'coulomb': myokit.units.C,
+    'dimensionless': myokit.units.dimensionless,
+    'farad': myokit.units.F,
+    'gram': myokit.units.g,
+    'gray': myokit.units.Gy,
+    'henry': myokit.units.H,
+    'hertz': myokit.units.Hz,
+    'item': myokit.units.dimensionless,  # Myokit does not have item unit
+    'joule': myokit.units.J,
+    'katal': myokit.units.kat,
+    'kelvin': myokit.units.K,
+    'kilogram': myokit.units.kg,
+    'liter': myokit.units.L,
+    'litre': myokit.units.L,
+    'lumen': myokit.units.lm,
+    'lux': myokit.units.lux,
+    'meter': myokit.units.m,
+    'metre': myokit.units.m,
+    'mole': myokit.units.mol,
+    'newton': myokit.units.N,
+    'ohm': myokit.units.R,
```

Comment:
Maybe just write `myokit.units.ohm` here. The `R` thing was a mistake I'll have to tidy up soon :-|

Reply:
done